### ISSUE TITLE
ENABLE_BITCODE on macosx causing carthage build failure

### ISSUE THREAD

**Issue by AndrewSB** [antitypical/Result#117]
Hey! I'm trying to build `Thomvis/BrightFutures` using Carthage, and I'm getting this error. I'm on Carthage 0.9.4.

```
*** Building scheme "BrightFutures-Mac" in BrightFutures.xcworkspace
Build settings from command line:
    ONLY_ACTIVE_ARCH = NO

=== BUILD TARGET Result-Mac OF PROJECT Result WITH CONFIGURATION Release ===

Check dependencies
target 'Result-Mac' has bitcode enabled (ENABLE_BITCODE = YES), but it is not supported for the 'macosx' platform

** BUILD FAILED **


The following build commands failed:
	Check dependencies
(1 failure)
A shell task failed with exit code 65:
** BUILD FAILED **


The following build commands failed:
	Check dependencies
(1 failure)
```

**Comment by AndrewSB** [comment]
I saw this was mentioned in #104 and #94

**Comment by ikesyo** [comment]
Yes, this is already resolved in 0.6.0-beta.5. BrightFutures should update the version.

**Comment by AndrewSB** [comment]
@ikesyo I think BrightFutures' version is updated, Carthage is  ```*** Checking out Result at "0.6.0-beta.6"```

**Comment by ikesyo** [comment]
It seems that BrightFutures uses submodules, the version in `Cartfile.resolved` and the submodule reference is 0.6.0-beta.4. BrightFutures itself is being built with the version.

**Comment by AndrewSB** [comment]
Thank you @ikesyo
